src/zulip.ts

```diff
@@ -131,7 +133,9 @@ export class Zulip extends EventEmitter {
 			new_user_input: 'false',
 			slim_presence: 'true',
 		};
-		return this.request('/users/me/presence', 'POST', params);
+		const response = await this.request('/users/me/presence', 'POST', params);
+		this.presenceUpdatedAt = new Date().getTime();
```

Comment:
I needed a way to expose the fact the Zulip module was setting the presence automatically. I went for this route.